cumulusci/core/config/universal_config.py

```diff
@@ -2,16 +2,23 @@
 from pathlib import Path
 
 from cumulusci.core.config import BaseTaskFlowConfig
-from cumulusci.core.config.project_config import BaseProjectConfig
+from cumulusci.core.config.project_config import (
+    BaseProjectConfig,
+    ProjectConfigPropertiesMixin,
+)
 from cumulusci.core.utils import merge_config
 from cumulusci.utils.yaml.cumulusci_yml import cci_safe_load
 
 __location__ = os.path.dirname(os.path.realpath(__file__))
 
 
-class UniversalConfig(BaseTaskFlowConfig):
+class UniversalConfig(BaseTaskFlowConfig, ProjectConfigPropertiesMixin):
```

Comment:
Oh, now I see why you created ProjectConfigPropertiesMixin. Well, give it a docstring to explain!

Reply:
Fixed now.